docs/WindowsBuild.md

```diff
@@ -126,8 +126,8 @@ set llvm_bin_dir="%swift_source_dir%/build/Ninja-DebugAssert/llvm-windows-amd64/
 - You may need to adjust the `SWIFT_WINDOWS_LIB_DIRECTORY` parameter depending on
   your target platform or Windows SDK version.
 ```cmd
-mkdir "%swift_source_dir%/build/Ninja-DebugAssert/swift-windows-amd64/ninja"
-pushd "%swift_source_dir%/build/Ninja-DebugAssert/swift-windows-amd64/ninja"
```

Comment:
@compnerd I can't remember if one needs to `cd` into the ninja dir to run cmake. Is `pushd`ing into the parent ok? 
If you're not sure I'll probably run this Monday/Tuesday next week and we can test then

Reply:
Id say test, but you can pass the path to the cmake invocation.

Reply:
I'll test on the next time I do a clean run